templates/homepage-blog-band.tmpl

```diff
@@ -0,0 +1,20 @@
+<div class="grid-wrapper homepage-blog-band">
+  <div class="width-12-12 width-12-12-m">
+    <div class="section-title">
+      <div class="section-top">
+        <img class="section-icon" src="/assets/images/blog.svg" />
+        <h2>{{ homepage.blog.title }}</h2>
+        <div class="section-view-more">
+          <a href="{{ homepage.blog.url }}">{{ homepage.labels.view_more }}&nbsp;<i class="fa fa-arrow-right"></i></a>
+        </div>
+      </div>
+      <p>{{ homepage.blog.intro }}</p>
+    </div>
+      {# shortcodes have to come from the markdown files, they don't work here, so the post_listing call is in pages/index.md #}
+      {# the homepage-blog-band class is also applied in index.md, otherwise a stray p element shows up #}
+      {% block content %}
+        {{ post.text() }}
+      {% endblock %}
```

Comment:
As mentioned in the previous comment, you can wrap this with a div to move it out of the index.html.